doc/rados/operations/pools.rst

```diff
@@ -188,7 +188,7 @@ consider removing them when you no longer need your pool::
 
 	ceph osd pool get {pool-name} crush_ruleset
 
-If the ruleset was "123", for example, you can check the other pools like so::
+If the ruleset ID was "123", for example, you can check the other pools like so::
```

Comment:
i think we are trying to use "ruleset" to note the "id", do you find somewhere else we are using "ruleset id" in the doc, and it's causing the confusion?